scripts/variantstore/gvs-overview.md

```diff
@@ -0,0 +1,165 @@
+| project_id | String describing the name of the Google project that contains the sample data. | String |
+| external_sample_names | Strings describing the unique sample IDs in the Terra data model. | Array of strings |
+| input_vcfs | Cloud paths to the sample gVCF files. | Array of files |
+| input_vcf_indexes | Cloud paths to the sample gVCF index files. | Array of files |
+| callset_identifier | String used as the name of the filter model and as the prefix to the names of the BigQuery extract tables and final joint VCF shards. | String |
+
+## Tasks and tools
+
+The [GVS workflow](https://github.com/broadinstitute/gatk/blob/rc-vs-483-beta-user-wdl/scripts/variantstore/wdl/GvsJointVariantCalling.wdl) imports additional WDL scripts that contain the workflow tasks. Links to these additional WDL scripts (subtasks/subworkflows) are provided below.
+
+Overall, the GVS workflow:
+
+1. Imports sample genomes.
+1. Trains the filtering model.
+1. Extracts VCF files.
+
+To see specific tool parameters used in each step of the workflow, select the WDL link in the table; then find the task and view the `command {}` section of the task in the WDL script.
+
+To view or use the exact tool software, see the task's Docker image, which is specified in the task `runtime {}` section as `docker: `.
+
+### 1. Import sample genomes
+
+To import sample genomes, the GVS workflow calls two subworkflows detailed below.
+
+#### A. GvsAssignIds
+
+The [GvsAssignIds subworkflow (alias = AssignIds)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsAssignIds.wdl) assigns GVS-specific IDs to each of the samples in the sample set which are used as an access pattern. The IDs enable easy access to specific sample data. GvsAssignIds imports an additional workflow, [GvsCreateTables](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsCreateTables.wdl).
+
+#### B. GvsImportGenomes
+
+The [GvsImportGenomes subworkflow (alias = ImportGenomes)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsImportGenomes.wdl) validates that sample gVCF files contain required annotations and loads samples into several tables in BigQuery. GvsImportGenomes imports an additional workflow, [GvsUtils (alias = Utils)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsUtils.wdl).
+
+### 2. Train the filtering model
+
+To train the filtering model, the GVS workflow calls two subworkflows detailed below.
+
+#### A. GvsCreateAltAllele
+
+The [GvsCreateAltAllele subworkflow (alias = CreateAltAllele)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsCreateAltAllele.wdl) splits alternate alleles and calculates additional annotations to be used for filtering. GvsCreateAltAllele imports an additional workflow, [GvsUtils (alias = Utils)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsUtils.wdl).
```

Comment:
It might be useful to list subworkflows separate from the description paragraph. The way it reads now, I almost expect the sentence about the additional workflow to be followed up with a description of what that subworkflow is doing.

We've done this for some of the other GATK pipelines like the WGS. We could use a similar technique here or try to a new way to break it up.

Reply:
Totally revamped the tasks and tools section of this doc. Let me know what you think!